### PR TITLE
gpu: Enable arm64 build of kernel

### PR DESCRIPTION
**Pull request by zvonkok**
For arm64 platform we need to have a different config then on x86_64, the offending setting CONFIG_PCI_MMCONFIG is not available on arm64.

Fixes: #6801

### CODE REVIEW

**Comment by fidencio** [comment]
@zvonkok, mind to rebase on `main`?

**Comment by fidencio** [comment]
> @zvonkok, mind to rebase on `main`?

Actually, hold on for a few hours and do a rebase, as more things related to the actions will be pushed soon.

File: tools/packaging/kernel/configs/fragments/gpu/nvidia.arm64.conf.in [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Support for loading modules.
+# It is used to support loading GPU drivers.
+CONFIG_MODULES=y
```
Comment (jongwu) [inline tools/packaging/kernel/configs/fragments/gpu/nvidia.arm64.conf.in:3]: So the gpu driver should be inserted into kata image?

Comment (zvonkok) [reply tools/packaging/kernel/configs/fragments/gpu/nvidia.arm64.conf.in:3]: Yes, so we can run unmodified containers, see here how it works: https://github.com/kata-containers/kata-containers/blob/main/docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md

**Comment by zvonkok** [comment]
Closing this in favour of https://github.com/kata-containers/kata-containers/pull/6949

**Pull request closed without merging**